### PR TITLE
Fix eco news dublicated tags and title field limit

### PR DESCRIPTION
**Pull request by TarasBeshlei**
Eco news with duplicated tags will not be created. Title limit changed to 170.

### CODE REVIEW

File: core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java [hunk 1]
```diff
@@ -76,6 +79,13 @@ public AddEcoNewsDtoResponse save(AddEcoNewsDtoRequest addEcoNewsDtoRequest,
         if (image != null) {
             toSave.setImagePath(fileService.upload(image).toString());
         }
+
+        Set<String> tagsSet = new HashSet<>(addEcoNewsDtoRequest.getTags());
+
```
Comment (antonbakalets) [inline core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: Could you please explain this if condition?

- `tagsSet `is created as a copy of `addEcoNewsDtoRequest.getTags()`
- so the size of `tagsSet `is the same as the size of `addEcoNewsDtoRequest.getTags()`
- so the if condition is always false
- so we have an unreachable statement?

Comment (TarasBeshlei) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: addEcoNewsDtoRequest.getTags() this is List which can contain duplicates of tags that not allowed. So I created a Set which will have smaller size than  addEcoNewsDtoRequest.getTags() if it contains duplicates

Comment (antonbakalets) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: Does it makes sense to save the Set without duplicates and do no return the error to user?  To make the correction for user on our own?

Comment (TarasBeshlei) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: In if condition I throw exception, so user will be informed about error. Also I thought about auto correction, but I made it as "Expected result" was required.

Comment (antonbakalets) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: Could you please give me a link to find the "Expected result"?

Comment (TarasBeshlei) [reply core/src/main/java/greencity/service/impl/EcoNewsServiceImpl.java:84]: https://github.com/ita-social-projects/GreenCity/issues/718

**Pull request merged**